src/RenderWebGL.js

```diff
@@ -290,6 +291,20 @@ class RenderWebGL extends EventEmitter {
         this.emit(RenderConstants.Events.NativeSizeChanged, {newSize: this._nativeSize});
     }
 
+    /**
+     * Notify Drawables whose skin is the skin that changed.
```

Comment:
I'd suggest adding a comment like this to avoid the possibility that someone might "optimize" this code and undo the performance improvement from this PR :)
```suggestion
     * Notify Drawables whose skin is the skin that changed. This loop may look slow but this method is expected to be
     * called rarely: only on user-driven edits, for example. A previous approach had Drawable registering for Skin's
     * alteration events but it turned out that registering and unregistering for those events took significant time.
```